www/views/tab-home.html

```diff
@@ -46,7 +46,7 @@
     </div>
 
     <div class="list card" ng-if="notifications[0] && recentTransactionsEnabled">
-      <a class="item item-icon-right item-heading" ui-sref="tabs.activity">
+      <a class="item item-icon-right item-heading" ui-sref="tabs.activity"></a>
```

Comment:
I think this tag here is closed by accident

Reply:
Yes, you're right.